admin/app/components/solidus_admin/ui/table/component.html.erb

```diff
@@ -11,7 +11,15 @@
   <% toolbar_classes = "h-14 p-2 bg-white border-b border-gray-100 justify-start items-center gap-2 visible:flex hidden:hidden" %>
 
   <div class="<%= toolbar_classes %>" data-<%= stimulus_id %>-target="scopesToolbar">
-    <%= render @tab_component.new(text: "All", 'aria-current': true, scheme: :secondary, href: "") %>
+    <form>
+      <%= render @tab_component.new(
+        text: "All",
+        'aria-current': true,
```

Comment:
Now that the implementation (using a `button` under the hood) is hidden, can't we just treat the way we set is as active as an implementation detail? I think it makes more sense as it was before, with the `active` and `disabled` params determining the state of the button.

Reply:
Active is still a state that cannot be controlled programmatically, so for that we still need another way to tap into it.

See https://developer.mozilla.org/en-US/docs/Web/CSS/:active#active_form_elements

Reply:
Yeah, sorry. For `active`, I meant `selected` (the current tab that is selected), not the activation on click.